plugins/cj-switch.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: cj-switch
+spec:
+  description: |
+    The list of the features:
+    * Invert the spec.suspend flag. 'True' turns to 'False' and vice versa
+    * Save current state to and restore state from file
+    * Overwrite spec.suspend flag
+    Possible filters:
+      * All namespaces
+      * Filter by label
+      * Filter by namespace
+    Example:
```

Comment:
Please move usages to -help of your plugin. These will be outdated.

Reply:
+1